frontstage_api/resources/secure_messaging/get_message.py

```diff
@@ -4,9 +4,10 @@
 from flask_restplus import Resource, reqparse
 from structlog import wrap_logger
 
-from frontstage_api import api, auth
+from frontstage_api import auth
 from frontstage_api.controllers import secure_messaging_controllers
 from frontstage_api.decorators.jwt_decorators import get_jwt
+from frontstage_api import secure_messaging_api
```

Comment:
Could this not be added to the auth import as is being imported from same place?